crates/vdo/examples/basic.rs

```diff
@@ -0,0 +1,25 @@
+use anyhow::{Error, Result};
+use vdo::{Stream, VdoBufferStrategy, VdoFormat};
+
+fn main() -> Result<()> {
+    let mut stream = Stream::builder()
+        .channel(0)
+        .format(VdoFormat::VDO_FORMAT_PLANAR_RGB)
+        .width(1920)
```

Comment:
would it be possible to add
resolution(1920, 1080)
instead? I know this a design flaw in the C API. but we could take the opportunity to improve the api, beause as it's designed right now it's possible to set only width... which would be ill-formed. so it would be better to remove the possibility of doing the wrong thing.

Reply:
Yes! That's a great idea to accept the width and height in one method call. I'll make the update.

Thank you for the suggestion.